### PR TITLE
updated to expo v33 and added support for react-native-web (expo-web)

### PR DESCRIPTION
**Pull request by neupsh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,8 +29,29 @@ $ shadow-cljs watch app
...
+For expo-web (react-native-web), use one of:
+```
+$ yarn start web
+# or
+$ yarn start web-only
+```
+This will run Expo DevTools at `http://localhost:19002/`
+This should also open the app automatically on your browser. If it does not open http://localhost:19006/ manually.
+
 Then use your editor of choice to hook up the REPL and such.
```
Comment (PEZ) [inline README.md]: This should probably stay under the **Or the Command line** heading.

File: README.md [hunk 2]
```diff
@@ -29,8 +29,29 @@ $ shadow-cljs watch app
 $ yarn start
 ```
 
+## React Native Web support
+For expo-web (react-native-web), use one of:
+```
+$ yarn start web
+# or
+$ yarn start web-only
+```
```
Comment (PEZ) [inline README.md]: I get this error when trying to start with web support:

```
$ yarn start web
yarn run v1.17.0
$ expo start web
Starting project at /Users/pez/Projects/rn-rf-shadow/web
Expo DevTools is running at http://localhost:19002
Can't read JSON file: /Users/pez/Projects/rn-rf-shadow/web/app.json
└─ Cause: Error: ENOENT: no such file or directory, open '/Users/pez/Projects/rn-rf-shadow/web/app.json'
No Expo configuration found. Are you sure this is a project directory?
```
Maybe you forgot to include that `web/app.json` file?

It's the same for `web-only`, but with then it complains about `web-only/app.json` not existing.

**Comment by PEZ** [comment]
Many thanks for upgrading the project! It's a general problem with templates/example projects like this that they fall behind.

File: README.md (hunk 1 above)
Comment (neupsh) [reply README.md]: I took a stab at it to make it less confusing. Please feel to edit it to make it better :)

File: README.md (hunk 2 above)
Comment (neupsh) [reply README.md]: Sorry, I had wrong instructions there. I did try it myself and made sure it works but had wrong instructions added in the readme. I have corrected it.

**Comment by PEZ** [comment]
Awesome. It works like a charm. Thanks again!

**Pull request merged**